folly/memory/Malloc.h

```diff
@@ -95,8 +95,8 @@ mallctlbymib(const size_t*, size_t, void*, size_t*, void*, size_t)
 
 // for malloc_usable_size
 // NOTE: FreeBSD 9 doesn't have malloc.h.  Its definitions
-// are found in stdlib.h.
-#if __has_include(<malloc.h>)
+// are found in stdlib.h and it errors when including malloc.h.
+#if __has_include(<malloc.h>) && !defined(__STDC__)
```

Comment:
When `__STDC__` is not defined to `1`, it is is sometimes undefined and it is sometimes defined to `0`, varying by platform.

Reply:
fixed.